en-us/install-instructions.md

```diff
@@ -95,17 +95,66 @@ For macOS or Linux [you need Java](https://adoptium.net/) to run the Fabric Inst
 
 A simple installer for vanilla launcher [is coming soon](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/110)!
 
+### [Pojav Launcher](https://github.com/PojavLauncherTeam/PojavLauncher) (Android)
+
+Fabulously Optimized can indeed be run on a mobile device! But there are _a lot_ of caveats:
+
+* The process has 20+ steps which you must follow carefully.
+* This process is for Android devices - phones and tablets. iOS devices _could_ work, but would also require even more steps and is untested.
+* Your device must be "mid-range" or "high-end" if you want decent FPS. 6+ GB RAM recommended.
+* Pojav Launcher supports Android 5 and up, but these instructions were tested on Android 12L, so some things may vary.
+* Your device will likely heat and consume a lot of battery as you play. It is not as device-optimized as Bedrock Edition.
+* Attempts have been made to improve performance, but depending on your device you may need to adjust video settings.
+* In the worst case, you may not get it running at all. Try again after a Pojav Launcher update.
+
+<details><summary><a href="https://play.google.com/store/apps/details?id=com.mojang.minecraftpe">I don't agree and just want to play Minecraft</a></summary></details>
+
+<details>
+  <summary>I agree and would like to see the instructions</summary>
+
+1. [Download Pojav Launcher](https://play.google.com/store/apps/details?id=net.kdt.pojavlaunch)
+   * Do not fall into scam websites and apps. The only official site for it is https://github.com/PojavLauncherTeam/PojavLauncher
+2. [Download Material Files](https://play.google.com/store/apps/details?id=me.zhanghai.android.files)
```

Comment:
Might want to rewrite the instructions with the system app, because that ensures it works without root and may even have easier access to the .minecraft folder
https://play.google.com/store/apps/details?id=com.marc.files